.evergreen/config.yml

```diff
@@ -378,14 +378,6 @@ axes:
         variables:
           TOPOLOGY: "replica_set"
 
-  - id: "jdk"
```

Comment:
JDK removed - as the gradle file now has tasks for running specific jdk versions.